lib-rca/src/main/java/com/facebook/openwifi/librca/models/LinkStats.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.models;
+
+import java.util.List;
+
+/** 
+ * Aggregation Statistics Model of InputStats. Aggregate by bssid, station and RadioConfig.
+ */
+public class LinkStats {
+    public static class RadioConfig {
+        public int channel;
+        public int channelWidth;
+        public int txPower;
+        public String phy;
+    }
+    public class Association{
```

Comment:
If this `Association` is the same as the `Association` in `InputStats` I think we should extract the `Association` class out separately and avoid duplicate code.